agent-local/ceph

```diff
@@ -21,7 +21,20 @@ def cephdf():
     cephdf    = check_output(["/usr/bin/ceph", "-f", "json", "df"]).replace('-inf', '0')
 
     s = json.loads(cephdf)
-    print("c:%i:%i:%i" % (s['stats']['total_bytes'], s['stats']['total_used_bytes'], s['stats']['total_avail_bytes']))
+    try:
+        ts = s['stats']['total_bytes']
+    except:
+        ts = s['stats']['total_space']
+    try:
+        tu = s['stats']['total_used_bytes']
+    except:
+        tu = s['stats']['total_used']
+    try:
```

Comment:
In my opinion i think the current way of outputting this is correct because it limits the values returned to LibreNMS via the agent tool. And you do not have to change LibreNMS it self everytime an update is done.

But yes, the variables are in the same units. I have it running for a while now, and it works.